src/main/frontend/components/search.css

```diff
@@ -1,17 +1,16 @@
 #search {
   > .inner {
     width: 100%;
+    max-width: 100%;
+
+    border-radius: 4px;
```

Comment:
@stdword Hello again, and thank you for contributing to the project! Could you provide an example of what those styles are fixing so we can verify it? Especially for the radius that seems to be superfluous.

Reply:
Any updates on this?

Reply:
@stdword ^^ Would be great to finish existing PRs before adding new ones